base/ca/src/main/java/org/dogtagpki/server/ca/cli/CACertRequestCLI.java

```diff
@@ -17,4 +19,21 @@ public CACertRequestCLI(CLI parent) {
 
         addModule(new CACertRequestImportCLI(this));
     }
+
+    public static void printCertRequestInfo(CertRequestInfo info) {
+
+        System.out.println("  Request ID: " + info.getRequestID().toHexString());
+        System.out.println("  Type: " + info.getRequestType());
+        System.out.println("  Status: " + info.getRequestStatus());
+        System.out.println("  Result: " + info.getOperationResult());
```

Comment:
From `Java 15` we have text blocks, so multiple return statements like this can now be formatted multiline strings:
```
        String details = """
                Request ID: %s
                Type: %s
                Status: %s
                Result: %s\
              """.formatted(
                        info.getRequestID().toHexString(),
                        info.getRequestType(),
                        info.getRequestStatus(),
                        info.getOperationResult());
        System.out.println(details);
```
I have preserved the 2 space indentation and no newline with this snippet but the syntax allows several options for controlling whitespace and newlines.

Maybe it is overkill for this use case but personally I think it is prettier - up to you :smile:

Reply:
This is a good idea, but I need to read more about text blocks, especially about the indentations, and my IDE doesn't recognize text blocks so it's giving me errors. Let's do this in a separate PR.

Reply:
Yeah the latest source compliance I have in my Eclipse (2021-03) is Java 15 so that is what I have enable. Seeing as how we have migrated to Java 17 I should probably look into upgrading Eclipse to a version that understands Java 17 exists!